ui/components/ui/loading-screen/index.scss

```diff
@@ -1,7 +1,7 @@
 .loading-overlay {
   top: 0;
   left: 0;
-  z-index: 51;
+  z-index: 55;
```

Comment:
Should we add a comment here to say what `55` is superceding in stack?

Reply:
Yep, added a comment :)